crypto/aes/asm/aes-s390x.pl

```diff
@@ -2219,6 +2254,69 @@ ()
 .size	AES_xts_decrypt,.-AES_xts_decrypt
 ___
 }
+
+################
+# void s390x_aes_gcm_blocks(unsigned char *out, GCM128_CONTEXT *ctx,
+#                           const unsigned char *in, size_t len,
+#                           const unsigned char *aad, size_t alen,
+#                           const AES_KEY *key, int enc)
+{
+my ($out,$ctx,$in,$len,$aad,$alen,$key,$enc) = map("%r$_",(2..9));
+$code.=<<___;
+.globl	s390x_aes_gcm_blocks
+.type	s390x_aes_gcm_blocks,\@function
+.align	16
+s390x_aes_gcm_blocks:
+.cfi_startproc
+	stm$g	$alen,$enc,7*$SIZE_T($sp)
+.cfi_rel_offset $alen,7*$SIZE_T
```

Comment:
Can we add .cfi-s separately?

Reply:
BTW, do you have a platform-specific DWARF addendum (or something)?

Reply:
Yes, i was thinking about removing before pushing. Added them for development purpose in some places where gdb would show "corrupted stack" otherwise and ended up adding them everywhere.

Theres no DWARF addendum for s390.